README.md

```diff
@@ -24,16 +24,16 @@ Una vez en el [sitio](https://tinchou.github.io/organizador-fiuba/), automática
 
 ### Reportando, sugiriendo
 
-- Se pueden reportar horarios incorrectos [acá](https://github.com/tinchou/organizador-fiuba/issues/new?title=[CARRERAS][CÓDIGO][CURSO]&labels=horarios).
-- Se pueden dejar sugerencias [acá](https://github.com/tinchou/organizador-fiuba/issues/new?labels=sugerencias).
+- Se pueden reportar horarios incorrectos [acá](https://github.com/lugfi/organizador-fiuba/issues/new?title=[CARRERAS][CÓDIGO][CURSO]&labels=horarios).
+- Se pueden dejar sugerencias [acá](https://github.com/lugfi/organizador-fiuba/issues/new?labels=sugerencias).
 
 ### Programando
 
 1. Hacer un fork del repositorio
-1. Hacer las modificaciones necesarias en la rama `gh-pages`, la cual automáticamente levanta el [sitio](https://tinchou.github.io/organizador-fiuba/)
+1. Hacer las modificaciones necesarias en la rama `gh-pages`, la cual automáticamente levanta el [sitio](https://lugfi.github.io/organizador-fiuba/)
 1. Crear un pull request para integrar los cambios en este repositorio
 
 # Creditos
 
 - Franco Spada es el autor original del organizador
-- [Martín Coll](https://github.com/tinchou) mantiene este repositorio y la versión web
+- [Martín Coll](https://github.com/lugfi) mantiene este repositorio y la versión web
```

Comment:
Agus le dio el OK en #12. Fixeemos los créditos y LGTM (let's get this
merged :p).

On Thu, Aug 4, 2016, 23:37 Fernando Danko <notifications@github.com> wrote:

> In README.md
> <https://github.com/lugfi/organizador-fiuba/pull/11#discussion_r73634942>:
>
> >  1. Crear un pull request para integrar los cambios en este repositorio
> >
> >  # Creditos
> >
> >  - Franco Spada es el autor original del organizador
> > -- [Martín Coll](https://github.com/tinchou) mantiene este repositorio y la versión web
> > +- [Martín Coll](https://github.com/lugfi) mantiene este repositorio y la versión web
>
> Mañana lo corrijo, tengo sueño Saludos, Fernando Danko El 4 de agosto de
> 2016, 21:56, Ezequiel <notifications@github.com> escribió:
> … <#m_3484453747846653817_>
> In README.md <#11 (comment)
> <https://github.com/lugfi/organizador-fiuba/pull/11#discussion_r73627933>>:
> > 1. Crear un pull request para integrar los cambios en este repositorio >
> > # Creditos > > - Franco Spada es el autor original del organizador > --
> [Martín Coll](https://github.com/tinchou) mantiene este repositorio y la
> versión web > +- [Martín Coll](https://github.com/lugfi) mantiene este
> repositorio y la versión web Por ahora dejalo a @tinchou
> <https://github.com/tinchou> <https://github.com/tinchou> como
> mantenedor, asi no se enoja (? Despues de que corrijas, apruebo el PR :) —
> You are receiving this because you were mentioned. Reply to this email
> directly, view it on GitHub <
> https://github.com/lugfi/organizador-fiuba/pull/11/files/ec7acdeaba65fc8842a3eb0fd86ab7f7c766abef..59f2552f3f979a7cdd2f2a39444ae5dca1a0f9c3#r73627933>,
> or mute the thread <
> https://github.com/notifications/unsubscribe-auth/AGsHAs2Wrur1OmwYzGkMYsBQpAmVsJTRks5qcopEgaJpZM4JawH->
> .
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/lugfi/organizador-fiuba/pull/11/files/ec7acdeaba65fc8842a3eb0fd86ab7f7c766abef..59f2552f3f979a7cdd2f2a39444ae5dca1a0f9c3#r73634942>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABeg9Mn57Mz1BjDlZvH7b6qxoVLLXJIPks5qcqHzgaJpZM4JawH->
> .
>